opae.spec.fedora

```diff
@@ -16,7 +16,7 @@ Requires:  systemd
 Requires:  tbb
 Requires:  hwloc
 Requires:  spdlog
-Requires:  libedit
+Requires:  edit
```

Comment:
We want libedit.  Remember from before?

Reply:
rpmlint error opae.x86_64: E: explicit-lib-dependency libedit
changed libedit -> edit

Reply:
package libedit and edit are different.  We discussed this before.  Please change it back to libedit.

Reply:
changed  back to libedit.